skymap_scanner/server/start_scan.py

```diff
@@ -328,6 +332,7 @@ async def scan(
     from_clients_queue: mq.Queue,
     nside_progression: NSideProgression,
     predictive_scanning_threshold: float,
+    realtime_format_version: str
```

Comment:
needs a trailing comma--this decreases diffs if a future change adds another arg